### ISSUE TITLE
Screen blinking in editor and resultset

### ISSUE THREAD

**Issue by lmayoni** [dbeaver/dbeaver#8288]
#### System information: 
- Operating system (distribution) and version: Linux Mint 19.2 (Tina)
- DBeaver version 7.0.1
- Additional extensions NO

#### Connection specification:
- Database name and version: mySql 7
- Driver name
- Do you use tunnels or proxies (SSH, SOCKS, etc)?  No

#### Describe the problem you're observing:
Screen is blinking all the time and turns sceen in gray. Its impossible to edit or write querys

**Comment by lmayoni** [comment]
https://streamable.com/dqgq2 link with a recorder video

**Comment by Piiit** [comment]
I have the same issue using Postgres databases, I think it has nothing to do which DB is used. It started with the last update to 7.0.1. I am on Linux Mint 19.3 Cinnamon

**Comment by facundokerbun** [comment]
Wow, I thought it was just me. I'm using Debian 9.12 Stretch and DBeaver version 7.0.1. Hope there will be a solution near in the future.

**Comment by serge-rider** [comment]
Could you post a short video/gif? 
The one above is not reachable.

**Comment by facundollampa** [comment]
Here's a gif that shows the issues I'm having.

![dbeaver-ce-v701](https://user-images.githubusercontent.com/25103921/78509549-15d67580-7765-11ea-8f7a-5ad182d94554.gif)

**Comment by Piiit** [comment]
Adding `export GTK_IM_MODULE=ibus` to my `~/.profile` before starting dbeaver 7.0.2 (from todays update) solved this issue for me.

I have seen this workaround here --> https://github.com/dbeaver/dbeaver/issues/4991

**Comment by facundollampa** [comment]
Hello, well, unfortunately that workaround did not work for me :disappointed:

**Comment by omar-gosaas** [comment]
This workaround didn't work for me either. This issue is occurring very frequently on my PC.
OS: Linux Mint
DBeaver: 7.0.2

**Comment by serge-rider** [comment]
Please check this workaround: https://bugs.eclipse.org/bugs/show_bug.cgi?id=517671#c75

**Comment by facundollampa** [comment]
> Please check this workaround: https://bugs.eclipse.org/bugs/show_bug.cgi?id=517671#c75

I appreciate the help, but just read the post and it will take me a while to understand this whole thing.
Regards

**Comment by lmayoni** [comment]
Fixed with last update.

**Comment by vaniusrb** [comment]
Hi folks! For me, the screen blinking still happening in the last version.

My setup:
OS: Linux Mint 19.3 Tricia x86_64
Kernel: 5.0.0-32-generic
Cinnamon 4.4.5
Intel i7-9750H (12) @ 2.600GHz
Intel UHD Graphics 630
NVIDIA NVIDIA Corporation Device 1c91
gtk-3-0:amd64 3.22.30

![Peek 2020-04-22 21-17](https://user-images.githubusercontent.com/11839284/80045970-7d1f5400-84df-11ea-9270-ee882e4e7508.gif)

**Comment by hellxz** [comment]
I have fix this problem by use command:
`sudo apt install ibus-gtk ibus-gtk3 ibus-clutter ibus-qt4 -y`

**Comment by rkharkar** [comment]
> I have fix this problem by use command:
> 
> ```shell
> sudo apt install ibus-gtk ibus-gtk3 ibus-clutter ibus-qt4 -y
> ```
> 
> then relogin your desktop, enjoy!

Worked for me. Thank you.

**Comment by dG94Cg** [comment]
OS: Arch Linux x86_64
Kernel: 5.11.4-arch1-1
Packages: 596 (pacman), 80 (guix-user), 2 (flatpak)
WM: awesome 
CPU: Intel i5-9500 (6) @ 4.400GHz 
GPU: Intel UHD Graphics 630 
PackageManager: Guix + Pacman
JDK: openjdk version "14" 2020-03-17 14.0+adhoc jdk14-bc54620a3848
GTK: gtk-engines	2.20.2	out	/gnu/store/vx91xpwmisz2mvdp48bp4h8hlyzdai7j-gtk-engines-2.20.2
InputMethod: Fcitx (from guix)
build dbeaver from git, problem still exists. ~~if i switch the theme into dark, it won't blink at sql editor~~, but still blink at table view layout. no further error log found. ibus won't fix the problem. 
> Dark Theme won't fix the blink, but hide it with same dark color. it does blink.

**Comment by hartois** [comment]
Fedora 33
XFCE
Fresh build from git: 21.0.3.202104110528
Bug exists
Workaround ```export GTK_IM_MODULE=ibus``` helps

**Comment by csonuryilmaz** [comment]
Thanks @hellxz, solved my problem on `Debian 9.13` with `dbeaver 7.3.5`. :tada:

**Comment by filippov70** [comment]
Operating System: Fedora Linux 35
KDE Plasma Version: 5.24.4
KDE Frameworks Version: 5.91.0
Qt Version: 5.15.2
Kernel Version: 5.18.10-100.fc35.x86_64 (64-bit)
Dbeaver 22.1.2.202207091909

_export GTK_IM_MODULE=ibus_ not solved problem

**Comment by hsnotebook** [comment]
I use fcitx5. Intall fcitx5-gtk and restart dbeaver, that solves the problem.

`pacman -S fcitx5-gtk`

**Comment by marceloudi** [comment]
> Here's a gif that shows the issues I'm having.
> 
> ![dbeaver-ce-v701](https://user-images.githubusercontent.com/25103921/78509549-15d67580-7765-11ea-8f7a-5ad182d94554.gif) [ ![dbeaver-ce-v701](https://user-images.githubusercontent.com/25103921/78509549-15d67580-7765-11ea-8f7a-5ad182d94554.gif) ](https://user-images.githubusercontent.com/25103921/78509549-15d67580-7765-11ea-8f7a-5ad182d94554.gif) [ ](https://user-images.githubusercontent.com/25103921/78509549-15d67580-7765-11ea-8f7a-5ad182d94554.gif)
> 
> I would like to add that this happens while using Classic Theme, but when changing it to "Dark" the issue is gone.

"Dark Theme" works here.

Linux mint XFCE

**Comment by alexandrehrt** [comment]
Dark Theme works here too
Fedora  XFCE

**Comment by blacklightpy** [comment]
Dark Theme also doesn't work
Void Linux Hyprland (Flatpak)

EDIT: Unchecking Enable Theming worked.

**Comment by MauricioOlson01** [comment]
It is not a perfect solution, but if you deactivate the themes in the dbeaver appearance option the bug stops occurring
